src/util/version.ts

```diff
@@ -57,11 +57,13 @@ export async function printVersion(
 
 export function variants(version: string) {
```

Comment:
I do believe the places where this function is called, there is also a SemVer object already, maybe it would make sense to add a parameter to have it passed in rather than parse the string value again?